themes/default/content/blog/resource-methods-for-pulumi-packages/index.md

```diff
@@ -0,0 +1,609 @@
+---
+title: "Introducing Resource Methods for Pulumi Packages"
+date: 2021-10-11T08:00:00-07:00
+draft: true
```

Comment:
If this post needs to go live when the PR is merged, we need to change the draft status.

```suggestion
draft: false
```